docs/projects/P2/1_teamprocess.md

```diff
@@ -0,0 +1,171 @@
+
+5. **Equitable Contribution & Conflict Resolution**<br/>
+What are the steps the team would take to address teammates who are contributing too little, and when will the team bring this up to the instructors? What are the steps to bring up and discuss potential teamwork issues? <br/><br/>
+The first thing the staff will ask the student when they mention that they are facing team issues is if they have followed the steps on their team contract.<br/>
+
+Feel free to seek the assistance of TAs and instructors in drafting this document. 
+
+!!! note
+    We will ask you to reference, reflect upon, and refine this document throughout this semester, and we will evaluate your team's process strategies and interactions through what you outline in this contract. Please ensure that everyone on your team thoroughly discusses each of the above sections and agree with the final decisions.
+
+Once you have completed the contract itself, have all members sign and date the document as an indicator that all members have read the document and agreed to uphold all outlined items. Then, save the file as a PDF and upload it to Gradescope. Only one team member needs to make the submission; they can then add the rest of the teammates as group members for a group submission. 
+
+We also **highly recommend** that you [bookmark](https://slack.com/help/articles/205239997-Pin-messages-and-bookmark-links) this document in your team’s Slack channel.
+
+
+### Project Planning (40 pts)
+
+Before your team jumps into development, you must first determine what features to implement. Schedule and hold an initial project planning meeting with your team to complete the steps outlined below.
+
+#### Functional Requirements - User Stories
+
+During this meeting, discuss potential functional requirements of this project. Consider what possible use cases may be for this system and what features it should have to fulfill those needs. 
+
+Then, document these functional requirements in the form of user stories that follow the guidelines discussed in lecture. All of your user stories should be in the format of "As a [role], I want [function], so that [value]".
+
+As a team, you should also come up with a prioritization ranking for each user story (i.e. how essential is this user story to the overall functionality of the application to your stakeholders?) 
+
+**You should come up with at least two user stories per student in your group.**
+
+!!! note "Formulating User Stories"
+    Consider what are the different types of users that will be interacting with your system (your stakeholders) and what features they would want to have that the current NodeBB does not provide. You may want to reference features in existing Q&A systems that may be desirable, or conduct interviews with people who could be a potential user of this system.
+
```

Comment:
Create a new GitHub Project board (click into `Projects`). Then create a new column at the leftmost called "User Stories", and then create another column to the right of it called "Backlog".

At this point in time, you should have 
"User Stories", "Backlog", "To-Do", "In Progress" and "Done".
Feel free to add more columns if needed during your development process